### PR TITLE
Improvement/updated nodejs_example.md

### PR DESCRIPTION
**Pull request by Sergioamjr**
For someone without package module knowledge, it's not so clear how to install the dependencies with yarn or npm, so, I updated `nodejs_example.md` file with the correct command line.

### CODE REVIEW

File: docs/nodejs_example.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+* Install the last version of [Chai](https://www.chaijs.com/) and Cucumber modules with [yarn](https://yarnpkg.com/en/) 
+```
+yarn add -D chai@latest cucumber@latest
+```
+
+* or with [npm](https://www.npmjs.com/)
+
+```
+npm i -D chai@latest cucumber@latest
+```
```
Comment (charlierudolph) [inline docs/nodejs_example.md]: Thoughts on leaving this as one bullet and the two snippets are together like:
```
yarn add -D chai@latest cucumber@latest
# or 
npm i -D chai@latest cucumber@latest
```
Also please indent the code snippet two spaces so it goes under the bullet

Comment (SergioJrDev) [reply docs/nodejs_example.md]: Done, @charlierudolph, pls, let me know if there is something else to improve in this file.

**Comment by aslakhellesoy** [comment]
Hi @Sergioamjr,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by aslakhellesoy** [comment]
Hi @Sergioamjr,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**